### DISCUSSION TITLE
As I execute the command make test-e2e-local  I'm receiving this error. can you please help me to resolve this error?

### DISCUSSION THREAD

**Discussion by dotsvm** [argoproj/argo-cd#15096] [Q&A] [answered]
Hello everyone, as I execute the command "make test-e2e-local"  I'm receiving this error. can you please help me to resolve this error?

![Screenshot from 2023-08-17 14-15-47](https://github.com/argoproj/argo-cd/assets/117660893/5bad8dc6-6ff5-488f-9864-9d9370d8a44a)

**Comment by crenshaw-dev** [comment, marked as answer]
Did you run `make start-e2e-local` first? Looks like Argo CD API server wasn't spun up.

**Reply by dotsvm** [reply]
Yes, sir, you are right. Now it's running perfectly.
